dev/generate_bom.py

```diff
@@ -202,9 +202,13 @@ def write_bom(self):
     output_yaml[VERSION] = toplevel_with_build
     self.__bom_file = '{0}.yml'.format(toplevel_with_build)
     self.write_bom_file(self.__bom_file, output_yaml)
-    self.publish_bom(self.__bom_file)
     output_yaml[VERSION] = 'nightly'
```

Comment:
where is "nightly" coming from and why is that hardcoded?
isnt a nightly build a specific context?